### ISSUE TITLE
Add ability to transfer snapshots between users

### ISSUE THREAD

**Issue by danielgolub** [digitalocean/api-v2#44]
I'm now building a project based on that capability.

Transferring snapshots not just between regions, but also between users.

Thanks in advance.

**Comment by brookemckim** [comment]
This will be added to the API, but no release date as of yet.

**Comment by phillbaker** [comment]
This has been released, please see the documentation here: https://developers.digitalocean.com/#transfer-an-image.

**Comment by danielgolub** [comment]
No, you sent documentation about sending images accross countries. 
Im talking about transferring the ownership to another user.

**Comment by fichek** [comment]
@phillbaker that has been out for a long time, but it's not the same thing.

**Comment by phillbaker** [comment]
Whoops, yes, my mistake. Thanks.

**Comment by christroutner** [comment]
It's approaching a year since this feature was requested. Any progress on creating an API call that can transfer snapshots between users?

**Comment by christroutner** [comment]
I just stumbled across [this feature proposal](https://digitalocean.uservoice.com/forums/136585-digitalocean/suggestions/7244493-user-to-user-transfer-via-api) on Digital Ocean. If you are a Digital Ocean user, be sure to go to that link and vote for this feature. Supposedly that will help the admins prioritize this as a feature.
